controllers/auth-controller.js

```diff
@@ -0,0 +1,69 @@
+const User = require("../models/user")
+const validationSession = require("../validation/validation-session")
+const validation = require("../validation/validation")
+
+function getSignup(req, res) {
+  const sessionErrorData = validationSession.getSessionErrorData(req, {
+    email: "",
+    confirmEmail: "",
+    password: ""
+  })
+
+  res.render("authentication/signup", {inputData: sessionErrorData})
+}
+
+function getLogin(req, res) {
+  res.render("authentication/login")
+}
+
+async function signup(req, res) {
+  const userData = req.body
```

Comment:
Alternative to replace these 4 lines with object expansion: `const {email, confirmEmail, password) = req.body`